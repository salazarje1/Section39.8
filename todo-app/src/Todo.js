import React from "react"; 


const Todo = ({ id, task, removeTask }) => {
    return (
        <div>
            <p>{task}</p>
            <button onClick={() => removeTask(id)}>Delete</button>
        </div>
    )
}


export default Todo;