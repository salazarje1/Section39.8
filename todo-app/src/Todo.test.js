import { render, fireEvent } from "@testing-library/react";
import Todo from "./Todo";

it('should render without crashing', () => {
    render(<Todo />);
})

it('should match snapshot', () => {
    const { asFragment } = render(<Todo />);
    expect(asFragment()).toMatchSnapshot();
})
