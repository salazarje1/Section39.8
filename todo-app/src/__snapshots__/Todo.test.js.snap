// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <p />
    <button>
      Delete
    </button>
  </div>
</DocumentFragment>
`;
